python_modules/dagster/dagster/_core/definitions/run_request.py

```diff
@@ -111,8 +114,50 @@ def __new__(
         )
 
 
+class IRunRequest(ABC):
+    pass
+
+
+@whitelist_for_serdes
+class BackfillDaemonRequest(
+    IRunRequest,
```

Comment:
I haven't encountered using a parent class and NamedTuple before. not sure if it is a python sin